### PR TITLE
Support both client token and id-token in script tag

### PR DESCRIPTION
**Pull request by sioked**
### Description
Currently with a client-token integration, we automatically enable vaulting for vaultable instruments. This change prevents that from happening if an id-token was also passed in to the sdk.

### Why are we making these changes? Include references to any related Jira tasks or GitHub Issues
We want to start promoting and transitioning vaulting functionality to use the id-token integration path, but the hosted-fields integration still requires a client token until it is replaced. This PR allows us to support both id-token and client-token data attributes so that hosted fields continues to work as expected, while SPB will use the id-token approach.

### Reproduction Steps (if applicable)
- Integrate using a vaulting test scenario with Hosted Fields and SPB
- When initializing, apply both the client-token and id-token
- Upon clicking the PayPal button, you should not see the `enableVault` mutation get called

### Screenshots (if applicable)
N/A

### Dependent Changes (if applicable)

N/A
### Groups who should review (if applicable)

❤️  Thank you!

### CODE REVIEW

File: src/middleware/vault-setup.js [hunk 1]
```diff
@@ -87,13 +88,20 @@ function isFundingSourceVaultable({ accessToken, fundingSource, clientID, mercha
...
-    clientID, merchantID, buyerCountry, currency, commit, intent, disableFunding, disableCard } : VaultAutoSetupEligibleProps) : ZalgoPromise<boolean> {
+    clientID, merchantID, buyerCountry, currency, commit, intent, disableFunding, disableCard, userIDToken } : VaultAutoSetupEligibleProps) : ZalgoPromise<boolean> {
 
     return ZalgoPromise.try(() => {
         if (!clientAccessToken) {
             return false;
         }
 
+        if (userIDToken) {
+          console.log('clientAccessToken and userIDToken');
```
Comment (elizabethmv) [inline src/middleware/vault-setup.js:99]:
```suggestion
```

**Review by elizabethmv — APPROVED** [review]
Looks good after one small fix

File: src/middleware/vault-setup.js (hunk 1 above)
Comment (sioked) [reply src/middleware/vault-setup.js:99]: Good catch, somehow my tired eyes missed that :-D

Comment (wsbrunson) [reply src/middleware/vault-setup.js]: this could be a good `calLog` though

**Review by wsbrunson — APPROVED** [review]

**Pull request merged**